ModularTegustation/fishing/code/fishing_items/fishing_refinery.dm

```diff
@@ -0,0 +1,56 @@
+/obj/structure/fish_refinery
+	name = "Fish Refinery"
+	desc = "A machine used to refine any fish into raw PE."
+	icon = 'ModularTegustation/fishing/icons/fishmachines.dmi'
+	icon_state = "fish_refinery"
+	anchored = TRUE
+	density = TRUE
+	resistance_flags = INDESTRUCTIBLE
+	var/refine_timer = 2 MINUTES
+	var/timeleft
```

Comment:
this variable is never used due to being a left-over from the old procs, so we can delete it
```suggestion
```